packages/fxa-auth-server/lib/senders/emails/partials/userLocation/index.txt

```diff
@@ -0,0 +1,12 @@
+<% const { city, stateCode, country } = location; %>
+  <% if (location.city) { %>
+    <% if (location.stateCode) { %>
+      location-all = "<%- location.city %>, <%- location.stateCode %> <%- location.country %> (estimated)"
```

Comment:
We destructured these above on L1. Should these just be `<% if (city) { %>` (similar to L6, L9, L11 below).

Reply:
I like how I did that for my own convenience and then I didn't even use it. 🥇